embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/session/EmbraceBackgroundActivityService.kt

```diff
@@ -50,10 +50,9 @@ internal class EmbraceBackgroundActivityService(
     override fun endBackgroundActivityWithState(timestamp: Long) {
         // kept for backwards compat. the backend expects the start time to be 1 ms greater
         // than the adjacent session, and manually adjusts.
-        val activity = backgroundActivity ?: return
         val message = stopCapture(
             FinalEnvelopeParams.BackgroundActivityParams(
-                initial = activity,
+                initial = backgroundActivity ?: return,
```

Comment:
What does putting a return here do?

Reply:
It does return from the method. However, I'll switch it back as it was clearer before